### PR TITLE
feat: screenshot protection

### PR DESCRIPTION
**Pull request by alexbarnsley**
## Summary

- Adds screenshot protection to prevent screen capture when configuration active in settings.

## Checklist

- [ ] Documentation _(if necessary)_
- [x] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

File: src/domains/dashboard/pages/Dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -23,6 +25,10 @@ export const Dashboard = ({ transactions, wallets, networks, portfolioPercentage
 	const activeProfile = useActiveProfile();
 	const history = useHistory();
 
+	React.useEffect(() => {
+		setScreenshotProtection(activeProfile?.settings().get(ProfileSetting.ScreenshotProtection) === true);
```
Comment (dated) [inline src/domains/dashboard/pages/Dashboard/Dashboard.tsx:29]:
```suggestion
		setScreenshotProtection(activeProfile?.settings().get(ProfileSetting.ScreenshotProtection) === false ? false : true);
```

This way screenshot protection will be enabled if `ProfileSetting.ScreenshotProtection` if falsy, but not `false`.

Comment (alexbarnsley) [reply src/domains/dashboard/pages/Dashboard/Dashboard.tsx]: fanks!

**Comment by faustbrian** [comment]
@alexbarnsley conflicts

**Comment by faustbrian** [comment]
@alexbarnsley conflicts

**Pull request merged**